src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -234,7 +234,7 @@ export default {
 		createDateDesc: 'Date/time this document was created',
 		documentType: 'Document Type',
 		editing: 'Editing',
-		expireDate: 'Remove at',
+		expireDate: 'Remove At',
```

Comment:
Just a heads up @AndyButland: screen readers read capitalization differently, so it will be more like: "Remove. At." than "Remove at".

Reply:
Thanks.  I didn't know that but makes sense now you point it out.

Trying to recall why I changed this - I think it was aligning with other labels where this is displayed, but can't see it now.  Will review and revert,